cmd/up.go

```diff
@@ -93,6 +93,11 @@ func ExecuteUp(ctx context.Context, wg *sync.WaitGroup, dev *model.Dev, namespac
 		return nil, err
 	}
 
+	sy, err := syncthing.NewSyncthing(namespace, d.Name, devList)
```

Comment:
I'm moving this up so that we fail if syncthing is missing before modifying the deployment